src/panels/PatientControlPanel.jsx

```diff
@@ -110,25 +128,31 @@ class PatientControlPanel extends Component {
                             <Col xs={3} lg={2} className='logo-title-column'>
                                 {this.renderFluxNotesLogo()}
                             </Col>
-                            <Col xs={5} md={4} lg={4} className="summary-header-column">
+                            <Col xs={4} md={4} lg={3} className="summary-header-column">
                                 {this.renderSummaryHeader()}
                             </Col>
-
+                            <Col xs={1} lg={1}>
+                                {this.renderSelectPatientArrow()}
+                            </Col>
                             <Col xs={4} md={5} lg={6}>
                                 {this.renderConditionSelectAndSearch()}
                             </Col>
                         </Row>
                     </Grid>
                 </Paper>
+                {this.isTablet && <PatientSelectionModal isModalOpen={this.state.isModalOpen} isTablet={this.isTablet} dataAccess={this.props.dataAccess} handleClose={this.handleClose} loadPatient={this.props.loadPatient}/>}
```

Comment:
[JA2] Small code styling comment, but it might be easier to read this PatientSelectModal component if we spread the props out onto their own lines, similar to the PatientSearch above it.